IMAGE.md

```diff
@@ -0,0 +1,45 @@
+# AutoHCK image creation
+
+AutoHCK is a tool for automating HCK/HLK testing, doing all the boilerplate steps in the process leaving you with simply choosing which driver you want to test on what os.
+
+## Manual installation
+
+Detailed instructions available at [HLK-Setup-Scripts](https://github.com/HCK-CI/HLK-Setup-Scripts).
+Images should be copied to the images directory as configured in the `config.json` file.
+
+## Automatic installation preparation
+
+Configure AutoHCK to have all information for building VM images. Edit the next files according to templates present in each file.
+
+1. `lib/engines/hckinstall/iso.json` - contains information about Windows ISO for each HCK/HLK platform. The following fields should be configured:
+   - **platform_name** - The platform name should be the same as in the `platforms.json` file;
+   - **path** - contains the path relative to `iso_path` option in config file to corresponding ISO image;
+   - **windows_image_names** - contains image name from `install.wim` file in ISO. To get a list of available images use the next commands:
+      * `dism /get-wiminfo /wimfile:D:\source\install.wim` on Windows;
+      * `wiminfo /mnt/source/install.wim` on Linux (wimlib is required);
+   - **product_key** - contains a valid key for the corresponding Windows image. Generic keys from MSDN can be used [KMS Client Setup Keys](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/jj612867(v=ws.11));
+
+2. `lib/engines/hckinstall/hckinstall.json` - contains the specific configuration for install engine. The following fields should be configured:
+   - **hck_setup_scripts_path** - path to HLK-Setup-Scripts repository;
+   - **answer_files** - list of the unattended answer file for automatic Windows installation. These files are part of HLK-Setup-Scripts repository;
+   - **studio_install_timeout** - timeout before the installation of the Studio VM is considered a failure;
```

Comment:
I would mark timeouts as the advanced configuration that should be changed only if the user understands why.